docs/sources/operations/query-acceleration-blooms.md

```diff
@@ -1,244 +1,54 @@
+This feature is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available.  No SLA is provided.
 {{% /admonition %}}
 
-Loki 3.0 leverages [bloom filters](https://en.wikipedia.org/wiki/Bloom_filter) to speed up queries by reducing the 
-amount of data Loki needs to load from the store and iterate through. Loki is often used to run “needle in a haystack” 
-queries; these are queries where a large number of log lines are searched, but only a few log lines match the [filtering 
-expressions]({{< relref "../query/log_queries#line-filter-expression" >}}) of the query. 
-Some common use cases are needing to find a specific text pattern in a message, or all logs tied to a specific customer ID.
+Loki leverages [bloom filters](https://en.wikipedia.org/wiki/Bloom_filter) to speed up queries by reducing the amount of data Loki needs to load from the store and iterate through.
+Loki is often used to run "needle in a haystack" queries; these are queries where a large number of log lines are searched, but only a few log lines match the query.
+Some common use cases are needing to find all logs tied to a specific trace ID or customer ID.
 
 An example of such queries would be looking for a trace ID on a whole cluster for the past 24 hours:
 
 ```logql
-{cluster="prod"} |= "traceID=3c0e3dcd33e7"
+{cluster="prod"} | traceID="3c0e3dcd33e7"
 ```
 
-Loki would download all the chunks for all the streams matching `{cluster=”prod”}` for the last 24 hours and iterate
-through each log line in the chunks checking if the string `traceID=3c0e3dcd33e7` is present.
+Without accelerated filtering, Loki downloads all the chunks for all the streams matching `{cluster="prod"}` for the last 24 hours and iterates through each log line in the chunks, checking if the [structured metadata][] key `traceID` with value `3c0e3dcd33e7` is present.
 
-With accelerated filtering, Loki is able to skip most of the chunks and only process the ones where we have a 
-statistical confidence that the string might be present. 
-The underlying blooms are built by the [Bloom Builder](#bloom-planner-and-builder) component
-and served by the new [Bloom Gateway](#bloom-gateway) component.
+With accelerated filtering, Loki is able to skip most of the chunks and only process the ones where we have a statistical confidence that the structured metadata pair might be present.
 
-## Enable Query Acceleration with Blooms
-{{< admonition type="warning" >}}
-Building and querying bloom filters are by design not supported in single binary deployment.
-It can be used with Single Scalable deployment (SSD), but it is recommended to
-run bloom components only in fully distributed microservice mode.
-The reason is that bloom filters also come with a relatively high cost for both building
-and querying the bloom filters that only pays off at large scale deployments.
-{{< /admonition >}}
+## Adding data to blooms
```

Comment:
Don't we still need the whole section about how to enable Blooms?
This section was updated in #13965 (August 27th) and #13997 (September 2), is it already out of date?

Reply:
I re-read the live content and it does seem correct to me; maybe I misunderstood what the team was saying we should do to the docs.

cc @chaudum Should I keep the content for the bloom components? Or was there a reason we wanted to remove it in the short term?

Reply:
Unless Blooms are enabled by default now, we should keep the content about how to enable it.

Reply:
@JStickler I brought back the old content, with some changes to make it accurate (some of it was outdated after all).

I also divided the document into a using/operating section, since the most important content for us at the moment is how to guide users to use query acceleration in Grafana Cloud.